plugin/hover.py

```diff
@@ -172,8 +191,11 @@ def code_actions_content(self) -> str:
         return "".join(formatted)
 
     def hover_content(self) -> str:
-        content = (self._hover.get('contents') or '') if isinstance(self._hover, dict) else ''
-        return minihtml(self.view, content, allowed_formats=FORMAT_MARKED_STRING | FORMAT_MARKUP_CONTENT)
+        contents = []
+        for hover_response in self._hover_responses:
+            content = (hover_response.get('contents') or '') if isinstance(hover_response, dict) else ''
+            contents.append(minihtml(self.view, content, allowed_formats=FORMAT_MARKED_STRING | FORMAT_MARKUP_CONTENT))
+        return ''.join(contents)
```

Comment:
Would it be better to have some kind of visual separator between the contents of multiple responses? Like `'<hr>'.join(contents)` (but the append content in the line above only if not empty).

Reply:
no `<hr>`:
![Screenshot from 2021-09-13 00-11-00](https://user-images.githubusercontent.com/22029477/133004436-65398020-2b1c-4197-9b2d-3e27e180afee.png)

with `<hr>`:
![Screenshot from 2021-09-13 00-12-18](https://user-images.githubusercontent.com/22029477/133004439-d7f3e833-42be-4a94-927e-6982ac634009.png)

This depends on personal preference.
I for example like the "no `<hr>`" option more.

At the end, this doesn't matter.
In practice, there will rarely be two servers that will provide hover responses for the given point in view.
Usually one server will return `None`. (but in theory it is possible that something like that happens)

Reply:
I like the "with `<hr>`" version a bit more, but I would leave it to you to decide. Or maybe someone can add a third opinion here.

Reply:
I prefer the `<hr>` version as well. Also, how would I know from which server the content came from in the popup?

Reply:
Also prefer `<hr>`. Depending on the content that two or more servers provide, it could get weird without some sort of divider.

> Also, how would I know from which server the content came from in the popup?

I also thought about that but then I've realized that it's probably not the kind of information that the user needs to know.

Reply:
I added the `<hr>` divider

>  I also thought about that but then I've realized that it's probably not the kind of information that the user needs to know.

+1